container_test.go

```diff
@@ -1426,6 +1426,20 @@ func TestExportContainerNoId(t *testing.T) {
 	}
 }
 
+func TestPutContainerArchive(t *testing.T) {
+	content := "File content"
+	in := stdinMock{bytes.NewBufferString(content)}
+	client := newTestClient(&FakeRoundTripper{status: http.StatusOK})
+	opts := PutContainerArchiveOptions{
+		Path:        "abc",
+		InputStream: in,
+	}
+	err := client.PutContainerArchive("a123456", opts)
+	if err != nil {
+		t.Errorf("PutContainerArchive: caugh error %#v while copying from container, expected nil", err.Error())
+	}
+}
```

Comment:
Can you improve this test by running some "assertions" on the request, using the FakeRoundTripper? Something like this one: https://github.com/fsouza/go-dockerclient/blob/master/image_test.go#L110-L116

Thanks!